src/erp/employee/EmployeeManager.java

```diff
@@ -2,31 +2,93 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.HashMap;  
+import java.util.Map;      
 
 public class EmployeeManager {
 
     private List<Employee> employees = new ArrayList<>();
+    private List<String> employeeNames = new ArrayList<>();  
+    private int employeeCount = 0;  
+    private static final String DEFAULT_DEPARTMENT = "General";  
+
+    public EmployeeManager() {
+        this.employeeCount = employees.size(); 
+        for (Employee emp : employees) {
+            employeeNames.add(emp.name);  
+        }
+        System.out.println("Employee Manager initialized with " + employeeCount + " employees.");
+    }
 
     public void addEmployee(Employee employee) {
         if (employee == null) {
             System.out.println("Invalid employee.");
             return;
         }
         employees.add(employee);
+        employeeNames.add(employee.name); 
+        employeeCount++; 
+        System.out.println("Employee added: " + employee.name);
     }
 
-    public void calculateAnnualBonus() { 
+    public void calculateMonthlySalary() {
         for (Employee emp : employees) {
-            double bonus = emp.salary * 0.1;  
-            System.out.println("Annual bonus for " + emp.name + ": " + bonus);
+            double salary = emp.salary / 12; 
+            System.out.println("Monthly salary for " + emp.name + ": " + salary);
         }
     }
 
-    public void listEmployeesByDepartment(String department) { 
+    public void calculateAnnualBonus() {
+        for (Employee emp : employees) {
+            double bonus;
+            if (emp.department.equals("Management")) {
+                bonus = emp.salary * 0.2; 
+                bonus = emp.salary * 0.1;
+            }
```

Comment:
How do you push something like this without a second glance at your code ? 
So according to you only management employees are allowed bonus? The rest don't matter ?
Regardless, I will be stating the obvious, you are missing an else statement after line 45. And if the company turns out to be one that doesn't care for others, mind removing the redundant assignment of bonus twice my lord.